src/main/java/com/microsoft/aad/msal4j/AzureCloudEndpoint.java

```diff
@@ -0,0 +1,26 @@
+package com.microsoft.aad.msal4j;
```

Comment:
Please add license header

Reply:
Added the license header, some javadocs with a link to [our wiki page on the national cloud stuff](https://github.com/AzureAD/microsoft-authentication-library-for-java/wiki/National-Clouds), and add some info about the enum on that wiki page.